README.md

```diff
@@ -249,21 +249,43 @@ if you make any changes to the ClusterRegistrar or HubConfig.  This will allow t
 
 # Using
 ## Import a user cluster into controller cluster
-1. Create and enter a new workspace in kcp
+1. Create and enter a new compute workspace in kcp. Compute workspace is the workspace where APIExport(schema) is created and user registers cluster.
```

Comment:
The Compute workspace is any workspace bound to the compute-apis APIExport where the user registers clusters.

probably *not* the workspace where the APIExport is created (though it could be)